portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -17,31 +17,59 @@
 import com.google.gson.Gson;
 import java.util.Date;
 import java.io.IOException;
+import java.util.Arrays;
 import java.util.ArrayList;
 import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
+// /** Servlet that returns some example content. TODO: modify this file to handle comments data */
+// @WebServlet("/data")
+// public class DataServlet extends HttpServlet {
```

Comment:
Nit: It's typically good practice to remove commented code.

Reply:
I'll go through and remove those